### ISSUE TITLE
Error when handling a cross()

### ISSUE THREAD

**Issue by mat-adamec** [scikit-hep/coffea#208]
I've had an analysis running successfully on a few datasets. I'm now trying to scale it up to include others, but Coffea seems not to like the most recent one I've added. I can't parse what the error is saying, aside from it coming from a point where I use the cross() function.

Unfortunately, I've been having trouble trying to reproduce it outside of my analysis, so I'm not sure what else to provide to help with debugging. I've pasted the error below, if there's anything else I can try to do to help pinpoint it, let me know.

```
_RemoteTraceback                          Traceback (most recent call last)
_RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/awkward/array/base.py", line 389, in _util_toarray
    return cls.numpy.frombuffer(value, dtype=getattr(value, "dtype", defaultdtype)).reshape(getattr(value, "shape", -1))
AttributeError: 'NoneType' object has no attribute '__buffer__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cvmfs/sft.cern.ch/lcg/releases/Python/3.6.5-54b64/x86_64-slc6-gcc62-opt/lib/python3.6/concurrent/futures/process.py", line 175, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/coffea/processor/executor.py", line 90, in _work_function
    out = processor_instance.process(df)
  File "<ipython-input-3-2185933ff257>", line 367, in process
    ee_jetcross = jets_2e[jets_2e['b_tag'] > loose_wp].cross(ee_lightjets[ee_lightjets.eta == ee_lightjets.eta.max()])
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/coffea/analysis_objects/JaggedCandidateArray.py", line 356, in cross
    outs = super(JaggedCandidateMethods, self).cross(other, nested)
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/awkward/array/jagged.py", line 1333, in cross
    out = self.JaggedArray.fromoffsets(argcross._offsets, self.Table.named("tuple", thyself._content[left], other._content[right]).flattentuple())
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/awkward/array/jagged.py", line 136, in fromoffsets
    offsets = cls._util_toarray(offsets, cls.INDEXTYPE, cls.numpy.ndarray)
  File "/home/hep/madamec/.local/lib/python3.6/site-packages/awkward/array/base.py", line 391, in _util_toarray
    if len(value) == 0:
TypeError: object of type 'NoneType' has no len()
"""

The above exception was the direct cause of the following exception:

TypeError                                 Traceback (most recent call last)
<ipython-input-16-c3f5ed9ed2aa> in <module>()
     12                                  executor=processor.futures_executor,
     13                                  executor_args={'workers':16},
---> 14                                  chunksize = 250000)

~/.local/lib/python3.6/site-packages/coffea/processor/executor.py in run_uproot_job(fileset, treename, processor_instance, executor, executor_args, chunksize, maxchunks)
    176     out = processor_instance.accumulator.identity()
    177     wrapped_out = dict_accumulator({'out': out, 'metrics': dict_accumulator()})
--> 178     executor(items, _work_function, wrapped_out, **executor_args)
    179     processor_instance.postprocess(out)
    180     if executor_args['savemetrics']:

~/.local/lib/python3.6/site-packages/coffea/processor/executor.py in futures_executor(items, function, accumulator, workers, status, unit, desc, **kwargs)
     68         futures = set()
     69         futures.update(executor.submit(function, item, **kwargs) for item in items)
---> 70         futures_handler(futures, accumulator, status, unit, desc)
     71     return accumulator
     72 

~/.local/lib/python3.6/site-packages/coffea/processor/executor.py in futures_handler(futures_set, output, status, unit, desc, futures_accumulator)
     45                 finished = set(job for job in futures_set if job.done())
     46                 for job in finished:
---> 47                     futures_accumulator(output, job.result())
     48                     pbar.update(1)
     49                 job = None

/cvmfs/sft.cern.ch/lcg/releases/Python/3.6.5-54b64/x86_64-slc6-gcc62-opt/lib/python3.6/concurrent/futures/_base.py in result(self, timeout)
    423                 raise CancelledError()
    424             elif self._state == FINISHED:
--> 425                 return self.__get_result()
    426 
    427             self._condition.wait(timeout)

/cvmfs/sft.cern.ch/lcg/releases/Python/3.6.5-54b64/x86_64-slc6-gcc62-opt/lib/python3.6/concurrent/futures/_base.py in __get_result(self)
    382     def __get_result(self):
    383         if self._exception:
--> 384             raise self._exception
    385         else:
    386             return self._result

TypeError: object of type 'NoneType' has no len()
```

**Comment by lgray** [comment]
@mat-adamec can you make a minimal example of this using JaggedCandidateArray?

**Comment by lgray** [comment]
@mat-adamec and does this problem persist with the latest versions of `coffea` and `awkward-array`?

**Comment by mat-adamec** [comment]
It appears to be running fine after an update!

**Comment by nsmith-** [comment]
I suspect this was fixed in #188
